app/views/admin/acls.py

```diff
@@ -47,11 +53,20 @@ def get_params(self):
     def _render_form(self):
         existing_acls = admin_acls_model.AdminPermission.get_for_repo(
             self.env.repo)
+        editable_acls = [
+            acl for acl in existing_acls if
+            self.env.user_admin_permission.compare_level_to(acl.access_level)
+            >= 0]
```

Comment:
just curious, would it help to define helper methods that encapsulate compare_to >= 0 / < 0, like ".is_at_least(acl...)" or ".is_less_than()"?

Reply:
If I find myself doing a lot more of this I probably will, but I think there's only one page where we have to vary behavior based on level, so it might be overkill. I'm going to keep it to one function for now.